React/CoreModules/RCTLogBox.mm

```diff
@@ -55,26 +55,26 @@ - (void)setSurfacePresenter:(id<RCTSurfacePresenterStub>)surfacePresenter
       }
 
       if (strongSelf->_bridgelessSurfacePresenter) {
-#if !TARGET_OS_OSX // [TODO(macOS GH#774)
+#if !TARGET_OS_OSX // [macOS]
         strongSelf->_view = [[RCTLogBoxView alloc] initWithFrame:RCTKeyWindow().frame
                                                 surfacePresenter:strongSelf->_bridgelessSurfacePresenter];
-#else
-        strongSelf->_view = [[RCTLogBoxView alloc] initWithSurfacePresenter:strongSelf->_bridgelessSurfacePresenter]; // TODO(macOS GH#774)
-#endif // ]TODO(macOS GH#774)
+#else // [macOS
+        strongSelf->_view = [[RCTLogBoxView alloc] initWithSurfacePresenter:strongSelf->_bridgelessSurfacePresenter]; // [macOS]
```

Comment:
Do we need `[macOS]` tags when we're already inside a broader `#else // [macOS` block?

Reply:
(Repeated throughout this file)

Reply:
Good catch! It's sometimes useful, but this looks like a case of redundancy.